static/app/components/devtoolbar/components/featureFlags/featureFlagItem.tsx

```diff
@@ -0,0 +1,100 @@
+import {Fragment, useContext, useState} from 'react';
+
+import AnalyticsProvider, {
+  AnalyticsContext,
+} from 'sentry/components/devtoolbar/components/analyticsProvider';
+import ExternalLink from 'sentry/components/links/externalLink';
+import {Cell} from 'sentry/components/replays/virtualizedGrid/bodyCell';
+import Switch from 'sentry/components/switchButton';
+
+import useConfiguration from '../../hooks/useConfiguration';
+import {inlineLinkCss} from '../../styles/link';
+import {panelInsetContentCss} from '../../styles/panel';
+import {smallCss} from '../../styles/typography';
+import type {FlagValue} from '../../types';
+
+import {useFeatureFlagsContext} from './featureFlagsContext';
+
+type FeatureFlag = {name: string; override: FlagValue; value: FlagValue};
+
+export default function FeatureFlagItem({flag}: {flag: FeatureFlag}) {
+  const {featureFlags} = useConfiguration();
+
+  return (
+    <Fragment>
+      <Cell css={[panelInsetContentCss, {alignItems: 'flex-start'}]}>
+        {featureFlags?.urlTemplate?.(flag.name) ? (
+          <ExternalLink
+            css={[smallCss, inlineLinkCss]}
+            href={featureFlags.urlTemplate(flag.name)}
+          >
+            {flag.name}
+          </ExternalLink>
+        ) : (
+          <span>{flag.name}</span>
+        )}
+      </Cell>
+      <Cell>
+        <FlagValueInput flag={flag} />
+      </Cell>
+    </Fragment>
+  );
+}
+
+function FlagValueInput({flag}: {flag: FeatureFlag}) {
+  if (
+    typeof flag.value === 'boolean' ||
```

Comment:
do we need the `typeof flag.value === 'boolean'` check?

Reply:
i was trying to keep the toolbar more agnostic to types, so we could extend it to do strings and numbers in the future